src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/AppealServiceTest.kt

```diff
@@ -308,6 +313,7 @@ class AppealServiceTest {
       val now = LocalDate.now()
 
       every { appealRepository.save(any()) } returnsArgument 0
+      every { assessmentService.createApprovedPremisesAssessment(any()) } returns mockk<ApprovedPremisesAssessmentEntity>()
```

Comment:
probably as well as setting up these stubs we also want to verify whether / when the appeal service reaches out to the assessment service? e.g.

- when appeal is accepted
  - `assessmentService.createApprovedPremisesAssessment()` is called with the `application`

- when appeal is rejected
  - `assessmentService.createApprovedPremisesAssessment()` is not called

Reply:
I've added these assertions in new tests.